website/content/docs/api-gateway/api-gateway-usage.mdx

```diff
@@ -307,3 +304,14 @@ spec:
 ```
 
 </CodeBlockConfig>
+
+
+### MeshService
+
+MeshService holds a reference to an externally managed Consul Service Mesh service, and can be used as a `backendRef` for a [`Route`](#route).
+
+| Parameter                       | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | Type    | Default         |
+| ------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- | --------------- |
+| `name`                      | Specifies the service name for a Consul service. It is assumed this service will exist in either the `consulDestinationNamespace` or mirrored Consul namespace from where this custom resource is defined, depending on the Helm configuration.
```

Comment:
```suggestion
| `name`                      | Specifies the service name for a Consul service. Consul assumes that this service will exist in either the `consulDestinationNamespace` or a mirrored Consul namespace from where this custom resource is defined, depending on the Helm configuration.
```
Is that right? That Consul is doing the assuming in this description?

Reply:
No. The point it is making is that the service needs to already exist. This isn't going to create the named service.